### ISSUE TITLE
"All Day" event checkbox is off screen rather than hidden when creating a VAMC event

### ISSUE THREAD

**Issue by fmccaf1** [department-of-veterans-affairs/va.gov-cms#10324]
## Describe the defect
When editor creates a new event, there is a checkbox "All Day" associated with the date and time area that is off screen/hard to find, but should actually be hidden until All Day events are supported.

## To Reproduce
1. Go to any event in the VHA section of the CMS
2. Click on edit, then zoom out, making the text smaller until you see a little checkbox appear out to the left.

## AC / Expected behavior

- [ ] All Day events field should be hidden
- [ ] Add PR link to #3250 so how to unhide is clear when the time comes.

## Screenshots
![Screen Shot 2022-08-18 at 10.24.23 AM.png](https://images.zenhubusercontent.com/629a1db0e70457da479e69ae/8e85d2d0-9fb7-4205-9138-bf36bf5aecc1)
![Screen Shot 2022-08-25 at 3 24 11 PM](https://user-images.githubusercontent.com/85581471/186779319-23e78412-aea7-4970-9004-799a82fa8819.png)

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by jilladams** [comment]
Ha! This field is supposed to be completely hidden until All Day Events are supported, epic https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3247, with a ticket in there, #3250 to unhide the All day event field. Which is currently not hidden. Rewriting ticket title to reflect the weirdness here.

**Comment by chri5tia** [comment]
![Screen Shot 2022-09-08 at 1 44 36 PM](https://user-images.githubusercontent.com/22764938/189222402-458700eb-86f2-4c21-8405-f9d0c68a4638.png)

![Screen Shot 2022-09-08 at 1 41 50 PM](https://user-images.githubusercontent.com/22764938/189222405-5b97c8b3-9a30-4a11-9191-c00d7cb7a3bf.png)

![Screen Shot 2022-09-08 at 1 47 02 PM](https://user-images.githubusercontent.com/22764938/189222790-57e4d7f6-a970-4376-b895-e3d04f7268a6.png)
![Screen Shot 2022-09-08 at 1 46 57 PM](https://user-images.githubusercontent.com/22764938/189222795-173f2283-7e73-495b-a9ff-90a6d7290292.png)

`display:block` for the `.allday-label` block is overriding any other `display:hidden` due to the `!important`:
![Screen Shot 2022-09-08 at 1 48 34 PM](https://user-images.githubusercontent.com/22764938/189223112-1381c164-b970-449d-8786-d4589353937a.png)

Summary for FE:

`recurring_events.css` has the following code:

```
  .allday-label {
    clear: both;
    display: block !important;      <--- this 
    float: none;
    font-weight: 400;
    left: -1000px;
    margin: 10px 0 0;
    position: absolute;
  }
```

Option 1:
Comment out the display block !important section above so that the `hidden.module.css` code can work:

```
/**
 * Hide elements from all users.
 *
 * Used for elements which should not be immediately displayed to any user. An
 * example would be collapsible details that will be expanded with a click
 * from a user. The effect of this class can be toggled with the jQuery show()
 * and hide() functions.
 */
.hidden {
  display: none;
}
```

Option 2:
Change the code noted above to show `display:hidden`. But for workflow reasons, the first option is probably better.

**Comment by chri5tia** [comment]
Part II: Drupal

- How is the "Smart Date" widget created?
Module:
![Screen Shot 2022-09-08 at 2 40 19 PM](https://user-images.githubusercontent.com/22764938/189230995-4a53164a-ccf6-4150-b11d-e664a936876f.png)

It is a contrib module. This is probably better addressed with the CSS only.

**Comment by chri5tia** [comment]
- [ ] Get context from @swirtSJW about this change, looks like it wasn't this line specifically though:

![Screen Shot 2022-09-08 at 2 52 56 PM](https://user-images.githubusercontent.com/22764938/189232619-28f6ebc3-2000-4cc3-97d4-d90412ff5ead.png)

**Comment by swirtSJW** [comment]
@chri5tia  if we are not using it, why not remove it from the form  in docroot/modules/custom/va_gov_events/src/EventSubscriber/EntityEventSubscriber.php  : alterEventNodeForm()

**Comment by chri5tia** [comment]
Well that's even better. My original thinking was to remove it all together, but I thought it was controlled by some contrib modules. My investigation never lead to this custom module, thank you!

**Comment by laflannery** [comment]
@swirtSJW  I believe (unless I'm looking at this incorrectly), this resolved my 508 issue #10131.  Let me know if not - but otherwise I will close my issue
